domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -252,6 +274,27 @@ class StoryProgressTestHelper @Inject constructor(
     )
   }
 
+/**
```

Comment:
Spacing is wrong here for KDoc--please fix.

Reply:
Done.